fossasia-gci-website.js

```diff
@@ -0,0 +1,209 @@
+            if(remote.url() === 'https://github.com/fossasia/gci15.fossasia.org.git' || remote.url() === 'https://github.com/fossasia/gci15.fossasia.org') { // "upstream" is set to FOSSASIA's repo
+              gciWebButton.mode += '-origin';
+              statusUpstreamIcon.className = 'material-icons gci-website__status-yes';
+              statusUpstreamIcon.innerHTML = 'done';
+              statusUpstreamText.className = 'gci-website__status gci-website__status-yes';
+              statusUpstreamText.innerHTML = 'is';
+              Git.Remote.create(repo, 'upstream', 'https://github.com/fossasia/gci15.fossasia.org.git');
+            }
+            repo.getRemote('origin')
+            .then(function(remote) {
+              if(remote.url() !== 'https://github.com/fossasia/gci15.fossasia.org.git' && remote.url() !== 'https://github.com/fossasia/gci15.fossasia.org') { // "origin" is set to a custom repo
+                gciWebButton.mode += '-upstream';
+                statusOriginIcon.className = 'material-icons gci-website__status-yes';
+                statusOriginIcon.innerHTML = 'done';
+                statusOriginText.className = 'gci-website__status gci-website__status-yes';
+                statusOriginText.innerHTML = 'is';
+              }
+              switch(gciWebButton.mode) {
+                // Everything is properly configured
+                case 'remote-upstream-origin':
+                case 'remote-origin-upstream':
+                  gciWebButton.disabled = true;
+                  gciWebButton.innerHTML = 'All set!';
+                  break;
+                // Either "upstream" or none of the remotes aren't configured
+                case 'remote':
+                case 'remote-upstream':
+                  gciWebButton.innerHTML = 'Configure "upstream"';
+                  break;
+                // The remote "origin" isn't correctly configured
+                case 'remote-origin':
+                  gciWebButton.innerHTML = 'Configure "origin"';
+                  break;
+              }
+            });
+          });
+        });
+    });
+  } else if(err.code !== 'ENOENT') {  // Something went wrong (excluding path not found)
+      console.log(err);
+    }
+  });
+}
+
+pathField.onclick = function(e) {
+  e.preventDefault();
+
+  // Set up dialog for opening repo's path
+  const options = {
+    title: 'Choose the directory where open/save the repo',
+    properties: [ 'openDirectory' ]
+  };
+
+  // Got the path from the dialog
+  ipcRenderer.on('openDialogReply', function(event, paths) {
+    if(paths !== null) {  // Check the dialog hasn't been cancelled
+      // Set the chosen path as pathField's value
+      pathField.value = paths[0];
+      // Update the statuses (changing the value won't trigger pathfield's oninput
+      // event)
+      updateStatuses();
+    }
+  });
+
+  // Display the dialog
+  ipcRenderer.send('openDialog', options);
+}
+
+pathField.oninput = function(e) {
```

Comment:
I left it there because you can still reach the text box's content by pressing <kbd>Tab</kbd> and modify it, so it still has some utility.  
However, it's not quite necessary, so we can remove it if you think it's not worth to have it there.